crates/bevy_reflect/src/type_registry.rs

```diff
@@ -101,8 +101,8 @@ impl TypeRegistry {
     }
 
     /// Registers the type `T`, adding reflect data as specified in the [`Reflect`] derive:
-    /// ```rust,ignore
-    /// #[derive(Reflect)]
+    /// ```ignore (Neither bevy_ecs nor serde "derive" is availible.)
```

Comment:
```suggestion
    /// ```ignore (Neither bevy_ecs nor serde "derive" are available.)
```